py-polars/polars/expr/datetime.py

```diff
@@ -57,6 +57,10 @@ def truncate(
             Every interval start and period length
         offset
             Offset the window
+
+            .. deprecated:: 0.20.19
+                This argument is deprecated and will be removed in the next breaking
+                release. Instead, chain `dt.truncate()` with `dt.offset_by()`.
```

Comment:
just for consistency
```suggestion
                release. Instead, chain `dt.truncate` with `dt.offset_by`.
```